### DISCUSSION TITLE
Multi-node, single-drive on each minio node config

### DISCUSSION THREAD

**Discussion by alexnixon2007** [minio/minio#17543] [Q&A] [answered]
Hey everyone. So, I have 4 VMs, each one has a disk mounted at /minio, which is supposed to be a minio storage. Every manual on how to make a cluster out of 4 nodes contains this step: you need to go to /etc/default/minio and add a string like this:

`MINIO_VOLUMES="http://s3node/{1...4}:9000/s3/disk{1...4}"
`
In my case, since I only have one drive per node, I add this string:

`MINIO_VOLUMES="http://s3-mnp-0{1...4}:9000/minio"
`
But when I do so, minio process fails to start. In fact, I only managed to start it replacing the string with this:

`MINIO_VOLUMES="/minio"
`
I also tried to use IP's instead of the hostnames (although name resolving seems to work fine on all 4 servers). Also I tried running all 4 nodes like so and configuring them in one cluster afterwards using mc, but I'm absolutely unsure if I'm doing it right. 

Minio service file for systemd is probably fine, since the service runs after all, but only if you only set it to use local drive only.

What am I doing wrong? Thanks a lot in advance.

**Comment by vadmeste** [comment]
@alexnixon2007 you just need to look at MinIO startup logs to see what is wrong, are you seeing anything ?

**Reply by alexnixon2007** [reply]
On my last restart attempt, I can see that connection is refused on port 9000 on three other nodes (which is no surprise, as minio service on those is also failed now), and this error, which I can't really understand how to fix:
ERROR Unable to initialize backend: /minio drive is already being used in another erasure deployment. (Number of drives specified: 4 but the number of drives found in the 1st drive's format.json: 1)
My format.json file, which is found at /minio/.minio.sys has this in it:

`{"version":"1","format":"xl-single","id":"8f5f7f17-305b-4830-af66-7353aaf70a92","xl":{"version":"3","this":"5a0e47ac-d8bc-428b-bba3-07c2918b16cb","sets":[["5a0e47ac-d8bc-428b-bba3-07c2918b16cb"]],"distributionAlgo":"SIPMOD+PARITY"}}
`

**Reply by klauspost** [reply, marked as answer]
When you change the configuration you must reformat or clear the drives.

**Reply by alexnixon2007** [reply]
That worked for me. Thank you very much for helping out a drowning newb.
